gamestonk_terminal/due_diligence/market_watch_api.py

```diff
@@ -15,6 +15,7 @@
 
 def sec_fillings(l_args, s_ticker):
```

Comment:
shouldn't it be sec_filings (1 l)

Reply:
It should, but I eat crayons for breakfast 🙃 ahah 

I'm not on my laptop now until later today, I'll merge it for now, and then fix this

Reply:
🤣  doesn't really matter lol